apps/atomy-q/API/tests/Feature/Api/V1/RequisitionVendorSelectionApiTest.php

```
@@ -0,0 +1,252 @@
+            'title' => 'Selection RFQ',
+            'status' => 'draft',
+            'owner_id' => $user->id,
+            'submission_deadline' => '2026-05-02 00:00:00',
+        ]);
+
+        return $rfq;
+    }
+
+    public function testItReplacesSelectedVendorsAndListsTheCurrentSelection(): void
+    {
+        $tenantId = (string) Str::ulid();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Str' in method 'testItReplacesSelectedVendorsAndListsTheCurrentSelection'. (undefined)

(StaticAccess)